src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -201,5 +202,35 @@ private void OnDescriptionViewLoaded(object sender, System.Windows.RoutedEventAr
             var browser = view.Browser;
             browser.ConsoleMessage += OnBrowserConsoleMessage;
         }
+
+        public string GetInstalledPackagesJSON()
+        {
+            string pkg = null;
+            dynamoWindow.Dispatcher.Invoke(new Action(() => pkg = LoadInstalledPackagesJSON()));
```

Comment:
We need to use dispatcher only for UI actions, in this case, you don't need to use dispatcher.  Only one method should be good, you can merge the other method with this.